lib/record_loader/library_type_loader.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
```

Comment:
Library type record loaders are nice and simple, and are pretty much just the template from record-loader itself.